docs/sections/fundamentals/basics.md

```diff
@@ -625,34 +719,61 @@ These are ideal for workflows that are not very computationally intensive and ca
 
 ```
 
+Just like we did for `aiida.calculations`, to see all available workflows you can run `verdi plugin list aiida.workflows`.
```

Comment:
Mmm I'm not sure what you mean with too much focus.

The usage cannot be removed completely since you need to at least use the `WorkflowFactory()` to load the `arithmetic.multiply_add`. It would be fairly easy to just...remove all mentions of "entry points" and just refer to the calculations / workflows as if the entry point was its name, but I don't think it is such a cognitive load and it is actually a good synonym to have available.

I removed a couple of mentions in the last modifications and together with taking out the first `verdi plugin list` command I think it has removed a lot of the negative impact of this.

Reply:
Looks a lot better already! :D Thanks!